examples/awsbraket.ipynb

```diff
@@ -128,13 +137,18 @@
    "metadata": {},
    "outputs": [],
    "source": [
+    "# Obtain and print the probabilies of the states\n",
     "prob_dict = eng.backend.get_probabilities(qureg)\n",
     "print(\"Probabilites for each of the results: \", prob_dict)"
    ]
   },
   {
    "source": [
-    "We can retrieve the result later on (of this job or a previously executed one) using the Task Arn provided when it was run. You have to remember the ammount of qubits involved in the job and the order you used."
+    "## Retrieve results form a previous execution\n",
+    "\n",
+    "We can retrieve the result later on (of this job or a previously executed one) using the Task Arn provided when it was run. You have to remember the ammount of qubits involved in the job and the order you used. Because in this case the job is not executed, but the result retrieved, we need to set up a mapping for the qubits.\n",
```

Comment:
```suggestion
    "We can retrieve the result later on (of this job or a previously executed one) using the task Arn provided when it was run. In addition, you have to remember the amount of qubits involved in the job and the order you used. The latter is required in case the job was not already executed since we need to set up a mapping for the qubits.\n",
```

Reply:
Sorry, my wording was horrible. This is not what I wanted to say. The mapping is required to retrieve the results whether the job was executed or not.

I propose:

`We can retrieve the result later on (of this job or a previously executed one) using the task Arn provided when it was run. In addition, you have to remember the amount of qubits involved in the job and the order you used. The latter is required since we need to set up a mapping for the qubits when retrieving results of a previously executed job.`

Reply:
In that case, I would suggest that we make some changes to the code in order to use the identity mapping automatically when the user does not specify any mapping.

See my new comments in the code for more precision.